### ISSUE TITLE
Call button not visible on Safari mobile

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#8008]
![9DB5A2A2-A868-42B8-B0BE-A92420B9A96B](https://user-images.githubusercontent.com/213943/192208595-5e431a05-fb84-481c-8ac4-ee7ea3ab18b0.jpeg)

## Browser

Safari from iPad OS 15.7

**Comment by nickvergessen** [comment]
That being said, also the leave call button seems to break sometimes:
![Bildschirmfoto vom 2022-09-27 15-42-00](https://user-images.githubusercontent.com/213943/192542952-374a67b4-cb7b-4125-a6e5-1aa7b5b63874.png)

**Comment by ChristophWurst** [comment]
Happens with FF 104.0.2 on Linux as well

**Comment by AndyScherzinger** [comment]
<img width="292" alt="2022-09-27 15_04_55 Talk - Nextcloud — Mozilla Firefox" src="https://user-images.githubusercontent.com/1315170/192544281-27509b66-1ce8-42b3-81e0-a11876e14b61.png">

Firefox 105.0.1 Win11Pro

**Comment by nickvergessen** [comment]
So that is actually another issue, so I will move it to https://github.com/nextcloud/spreed/issues/8030

**Comment by nickvergessen** [comment]
First thing to try is removing this
https://github.com/nextcloud/spreed/pull/8031/files#diff-bc6ee841a7815ccb651df88720f57def8e7da3b24263a946a388c8e3b60202cdR313

**Comment by marcoambrosini** [comment]
can't reproduce neither on ios 15 nor 16

<img width="1013" alt="Screenshot 2022-09-29 at 11 48 44" src="https://user-images.githubusercontent.com/26852655/192999956-23c9607b-e185-4fc4-b27e-f9213c796050.png">

**Comment by nickvergessen** [comment]
I just noticed that it also happens for the "Join call" button in the chat view. So something seems to be wrong with that component in general and it's not caused by the topbar.

**Comment by nickvergessen** [comment]
So the origin of the issue is the tooltip.

As soon as:
https://github.com/nextcloud/spreed/blob/f3f250f9591e14ec33083e60adbe149bdf797771/src/components/TopBar/CallButton.vue#L29
Is replaced with:
```js
content: 'startCallToolTip',
```

Digging deeper I found that the problem origins from the computed we use there. Using a computed is actual fine.

More digging there detected browser version in is `undefined` so everything breaks.

Turns out iPad OS safari has a `,` in it's version leading to an upstream bug:
https://github.com/faisalman/ua-parser-js/issues/599
